experiments/gamma_experiments.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from autograd import numpy as np
+from lifelines.fitters import KnownModelParametericUnivariateFitter, ParametericUnivariateFitter
+from lifelines.utils.gamma import gammainc, gammaincc
+
+
+class GG(ParametericUnivariateFitter):
+
+    _fitted_parameter_names = ["alpha_", "lambda_", "rho_"]
+    _bounds = [(0.0, None), (0.0, None), (0.0, None)]
+
+    def _cumulative_hazard(self, params, times):
+        times = times / 100
+        alpha_, lambda_, rho_ = params
+        lg = np.clip(gammaincc(alpha_ / rho_, (times / lambda_) ** rho_), 1e-20, 1 - 1e-20)
+        return -np.log(lg)
+
+
+from lifelines.datasets import load_waltons
+
+df = load_waltons()
+
+gg = GG()
+gg.fit(df["T"], df["E"])
+
+gg.print_summary()
+
+
+class LogGammaFitter(ParametericUnivariateFitter):
```

Comment:
@cristobaltapia here's an implementation of your model using a slightly more stable gamma module (see `gamma.py` file below). Note that _scaling time to be small_ is really important to stability, example: https://github.com/CamDavidsonPilon/lifelines/pull/766/files#diff-2ba6fa67c351fd9251fde3426fbcae29R45

Reply:
Wow, this is great! Thanks! I will try this tomorrow :D